tests/fixtures/db_fixtures.py

```diff
@@ -0,0 +1,67 @@
+import os
+import pytest
+from ros.lib.app import app, db
+from sqlalchemy_utils import database_exists, create_database, drop_database
+from ros.lib.models import RhAccount, System
```

Comment:
Just a small  query - don't we need a test for models?

Reply:
No, There is no need to test the models.